### ISSUE TITLE
Using multiple inputs for `nonlinParamSys` causes crash

### ISSUE THREAD

**Issue by dilr** [TUMcps/CORA#38]
When trying to use the `nonlinParamSys` class with multiple time varying inputs, I get the error:

```
Incorrect number or types of inputs or outputs for function 'dim'.

Error in equalDimCheck (line 50)
        ind_dim = dim(obj)==dim(S);

Error in  *  (line 115)
    equalDimCheck(Z,M);

Error in inputSolution (line 66)
inputSetTrans = intM*zonotope(vTrans);
```

I've cut out the stack traces above the multiplication which causes the error.

I've attached a file which causes the error. (rename to .m to get around GitHub upload limitations) Note that a different error is caused if only one parameter is used instead of two. This error also stops occurring if there is only one RHS equation or input, or if the number of RHS equations and input are equal. I'm running CORA 2022 stable with MATLAB 2022b on MacOS.

[param_test.txt](https://github.com/TUMcps/CORA/files/10429167/param_test.txt)

**Comment by dilr** [comment]
I've investigated the issue myself to some extent. From what I can see, in `@nonlinParamSys/linearize`, when the `linearParamSys` is created, `B` is set to 1 with the comment that the matrix is already applied to the uncertain input. this is true of the `U` parameter, but not of the `uTrans` parameter in the time invariant code branch. Thus when `inputSolution` attempts to multiply `intM`  by `vTrans` (`vTrans = B*uTrans`), the dimensions don't match up.

Since `@nonlinParamSys/linearize` does use `uTrans` to calculate the linearized value of `U`, I think it would be appropriate to set the linearized `uTrans` to 1. However, I don't understand the algorithm well enough to say that for certain.

**Comment by wetzlingerm** [comment]
Thanks for catching this error! Your analysis was indeed correct. The issue has been resolved in v2022.0.12.
